### ISSUE TITLE
Danielle experienced performance issue

### ISSUE THREAD

**Issue by LucyPilot** [LucyPilot/Issues-Tracking#36]
She got couple Lucy error messages on the left panel and asked her to refresh the page. She did refresh the page and then she got logged out. She reported this issue on 5/10/2019 Friday stand-up meeting. But she had experienced this early that week.

**Comment by LucyPilot** [comment]
We communicated to the pilot team that they need to = record the issue and report earlier.

**Comment by LucyPilot** [comment]
I experienced this issue as well I believe. I might forget to log out last Friday. Today (Monday), I tried my session, it is still showing the previous questions I asked. Then I asked a new question, It just keeps spinning at 90% and asked me to refresh the page. But refreshing doesn't help. I need to logout and then the session is cleared. Previous questions are gone. And the new question I asked in the previous bad session was not captured as the questions I asked before. This is related to auto-logout users when they close the window or at least by the end of the day. I didn't use Lucy for 2 days. But my previous session never expired.
